airflow/contrib/hooks/bigquery_hook.py

```diff
@@ -1114,18 +1184,18 @@ def get_schema(self, dataset_id, table_id):
         :param table_id: the table ID of the requested table
         :return: a table schema
         """
-        tables_resource = self.service.tables() \
-            .get(projectId=self.project_id, datasetId=dataset_id, tableId=table_id) \
-            .execute()
+        tables_resource = self.service.tables().get(
+            projectId=self.project_id, datasetId=dataset_id,
+            tableId=table_id).execute()
         return tables_resource['schema']
 
     def get_tabledata(self, dataset_id, table_id,
                       max_results=None, selected_fields=None, page_token=None,
                       start_index=None):
         """
-        Get the data of a given dataset.table and optionally with selected columns.
-        see https://cloud.google.com/bigquery/docs/reference/v2/tabledata/list
-
+        Get the data of a given dataset.table and optionally
+        with selected columns. see:
+        https://cloud.google.com/bigquery/docs/reference/v2/tabledata/list
```

Comment:
Missing newline between description and params list

Reply:
fixed